### DISCUSSION TITLE
Issue with consume

### DISCUSSION THREAD

**Discussion by para-dise** [Minestom/Minestom#1180] [Help] [answered]
It seems that whenever I call ItemStack#consume its not actually removing an item from the player, maybe im just doing it wrong?

My code is

```java
handler.addListener(PlayerUseItemEvent.class, event -> {
            if(event.getItemStack().getMaterial() == Material.CLOCK) {
                 event.getItemStack().consume(1);
            }
        });
```

And no item is being removed from the player when he clicks the clock

**Comment by breitwan** [comment, marked as answer]
ItemStack#consume method returns new ItemStack instance, so you have to give new item to player
